app/services/browser_support.rb

```diff
@@ -1,10 +1,12 @@
 class BrowserSupport
   @cache = LruRedux::Cache.new(1_000)
 
+  # rubocop:disable Layout/LineLength
   BROWSERSLIST_TO_BROWSER_MAP = {
-    and_chr: ->(browser, version) { browser.android? && browser.chrome?(version) },
+    and_chr: ->(browser, version) { browser.android? && !browser.platform.android_webview? && browser.chrome?(version) },
     and_uc: ->(browser, version) { browser.android? && browser.uc_browser?(version) },
-    chrome: ->(browser, version) { browser.chrome?(version) },
+    android: ->(browser, version) { browser.platform.android_webview? && browser.send(:detect_version?, browser.version, version) },
```

Comment:
I don't love the `send` here, but `android_webview?` doesn't accept parameters like the other browser matchers. Internally, this method is what the `browser` gem uses for parameterized methods ([source](https://github.com/fnando/browser/blob/fa4f685482c315b8d0b896d0c82bfe667ab3e2bb/lib/browser/base.rb#L145-L148)). 

An alternative implementation could be to use [the same underlying core logic](https://github.com/fnando/browser/blob/fa4f685482c315b8d0b896d0c82bfe667ab3e2bb/lib/browser/detect_version.rb#L12-L13), but this risks some drift in the logic, and is more verbose.